### PR TITLE
Navigate to url

### PR DESCRIPTION
**Pull request by kailash-choudhary-thoughtspot**

### CODE REVIEW

File: answer/templates.html [hunk 1]
```diff
@@ -17,6 +17,10 @@
 </script>
 <!-- runtimeFilters -->
 <script type="text/template" id="runtimeFilters">
-     runtimeFilters: [],
+     runtimeFilters: [{
+          columnName: 'color',
```
Comment (jbhanu-thoughtspot) [inline answer/templates.html]: Can you change this to `columName1`? and value red to `value1`. Do the same with all the other occurences. You can maybe add `// eg: color` at the line end instead

File: answer/templates.html [hunk 2]
```diff
@@ -17,6 +17,10 @@
...
+          operator: RuntimeFilterOp.EQ,
+          values: [ 'red' ]
+          }],
```
Comment (jbhanu-thoughtspot) [inline answer/templates.html]: Can you fix this spacing?

File: answer/templates.html [hunk 3]
```diff
@@ -17,6 +17,10 @@
 </script>
 <!-- runtimeFilters -->
 <script type="text/template" id="runtimeFilters">
-     runtimeFilters: [],
+     runtimeFilters: [{
+          columnName: 'color',
+          operator: RuntimeFilterOp.EQ,
```
Comment (jbhanu-thoughtspot) [inline answer/templates.html:22]: Is `RuntimeFilterOp` already part of imports?

File: app/templates.html [hunk 4]
```diff
@@ -15,8 +15,17 @@
 <script type="text/template" id="showNavBar">
      showPrimaryNavbar: true,
 </script>
+<!-- navigateToUrl-->
+<script type="text/template" id="navigateToUrl">
+     path: "",
+     // eg pinboard/[pinboardId] or insights/results
```
Comment (jbhanu-thoughtspot) [inline app/templates.html]: Keep the comment above

File: pinboard/templates.html [hunk 5]
```diff
@@ -13,6 +13,10 @@
 </script>
 <!-- runtimeFilters -->
 <script type="text/template" id="runtimeFilters">
-     runtimeFilters: [],
+     runtimeFilters: [{
+          columnName: 'color',
+          operator: RuntimeFilterOp.EQ,
```
Comment (jbhanu-thoughtspot) [inline pinboard/templates.html:18]: Make sure to add `RuntimeFilterOp` to all the params

**Review by jbhanu-thoughtspot — CHANGES_REQUESTED** [review]

File: answer/templates.html (hunk 3 above)
Comment (kailash-choudhary-thoughtspot) [reply answer/templates.html:22]: yes

**Pull request closed without merging**